pkg/operator/certrotationcontroller/certrotationcontroller.go

```diff
@@ -50,6 +51,7 @@ func NewCertRotationController(
 	kubeInformersForNamespaces v1helpers.KubeInformersForNamespaces,
 	eventRecorder events.Recorder,
 	day time.Duration,
+	refreshOnlyWhenExpired bool,
```

Comment:
make two `New***` method variants instead of this bool please. This method can then be internal with a bool arg. That way it's very obvious and intentional.

Reply:
done